canvas2D/src/Engine/babylon.group2d.ts

```diff
@@ -912,23 +914,12 @@
             // For now we only support these property changes
             // TODO: add more! :)
             switch (prop.id) {
-                case Prim2DBase.actualScaleProperty.id:
                 case Prim2DBase.actualPositionProperty.id:

```

Comment:
I think the position need not multiple scale. 
like i modify in _allocateGroupCache too.